src/java/org/apache/nutch/crawl/LinkDb.java

```diff
@@ -56,11 +61,11 @@
   public static final String CURRENT_NAME = "current";
   public static final String LOCK_NAME = ".locked";
 
-  private int maxAnchorLength;
-  private boolean ignoreInternalLinks;
-  private boolean ignoreExternalLinks;
-  private URLFilters urlFilters;
-  private URLNormalizers urlNormalizers;
+  private static int maxAnchorLength;
+  private static boolean ignoreInternalLinks;
+  private static boolean ignoreExternalLinks;
+  private static URLFilters urlFilters;
+  private static URLNormalizers urlNormalizers;
```

Comment:
Similar to the situation in Generator: maybe it's better to make them instance variables in LinkDbMapper.